### ISSUE TITLE
Not everything filters on database

### ISSUE THREAD

**Issue by jasonumiker** [ankane/pghero#42]
The Queries section filters on DB with the WHERE below with but other things like long running queries on the main page do not. We run a large shared instance where we'd like to prevent individual DBs (one allocated to each service/team) from seeing/killing the queries of other DBs.
         
```
 INNER JOIN
              pg_database ON pg_database.oid = pg_stat_statements.dbid
            WHERE
              pg_database.datname = current_database()
```

**Comment by ankane** [comment]
Nice catch @jasonumiker.  PgHero should filter by DB for all methods, except for connections (since this is really a server limit), but it should differentiate between your connections and other connections).  Feel free to submit a pull request, or I'll get to it when I have the time.

**Comment by dhcrzf** [comment]
I believe I have addressed this issue with the following modification.

- added "AND datname = current_database()" to the WHERE clause for methods 

```
- PgHero.Methods.Kill.kill_all
- PgHero.Methods.Queries.running_queries
- PgHero.Methods.Queries.long_running_queries
- PgHero.Methods.Queries.locks
```

@ankane Can I assume from your comment that you would like the connection counts in the Overview display and Connections tab modified to differentiate between server connections and current_database connections? Something like "Healthy number of connections 3 (2 to current database)"?

**Comment by ankane** [comment]
Fixed with #59 - thanks @dhcrzf!
